content/en/docs/concepts/overview/object-management-kubectl/kustomization.md

```diff
@@ -1,24 +1,31 @@
 ---
-title: Declarative Management of Kubernetes Objects Using Kustomization
+title: Declarative Management of Kubernetes Objects Using Kustomize
 content_template: templates/concept
 weight: 40
 ---
 
 {{% capture overview %}}
 [Kustomize](https://github.com/kubernetes-sigs/kustomize) is a standalone tool
 to customize Kubernetes objects
-through a [kusotmization file](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/kustomization.yaml). 
+through a [kustomization file](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/kustomization.yaml). 
 Since 1.14, Kubectl also
 supports the management of Kubernetes objects using a kustomization file.
-When dealing with Resources or objects from kustomization, the flag
-`--kusotmize` or `-k` need to be used in Kubectl commands.
+To view Resources managed inside a directory containing a kustomization file, you can run following command
+```shell
```

Comment:
Possible edit: To view Resources found in a directory containing a kustomization file, run the following command:

Reply:
updated